api/spec/v1.yaml

```diff
@@ -1350,6 +1352,18 @@ components:
           format: int32
           description: Number of transactions in the block.
           example: 17
+        gas_limit:
+          allOf: [$ref: '#/components/schemas/TextBigInt']
+          description: The gas limit for the block.
```

Comment:
```suggestion
          description: The gas limit for the block. A gas limit of 0 means that the max block gas was unlimited. 
          Blocks from Cobalt and Damask always have a gas limit of 0 because the consensus parameters were static during those periods.
```

Reply:
> Blocks from Cobalt, Damask, and early Eden had no gas limit as their sizes were only restricted by byte size until a mid-Eden upgrade introduced a gas limit.

^ this is a bit more factually correct.